public/random/generic.html

```diff
@@ -37,38 +40,41 @@ <h1>Index</h1>
     <li><a href="#dormont">Doors in Dormont</a></li>
     <li><a href="#flowers">Flowers</a></li>
     <li><a href="#hole">Holes</a></li>
-    <li><a href="#door">Locked doors</a></li>
+    <li><a href="#door">Locked openphrase doors</a></li>
     <li><a href="#save">Save points</a></li>
     <li><a href="#tears">Tears</a></li>
     <li><a href="#torch">Torches</a></li>
     <li><a href="#equipment">Refinding equipment</a></li>
     <li><a href="#act5">ACT 5</a>
         <ul>
-            <li><a href="#items5">Items</a></li>
+            <li><a href="#item5">Items</a></li>
+            <li><a href="#tears5">Tears</a></li>
             <li><a href="#save5">Save points</a></li>
             <li><a href="#nothing">Nothing else</a></li>
         </ul>
     </li>
 </ul>
 
+<hr>
 <br>
 
 <h1 id="bird">Birds</h1>
 <p><b>Source:</b> Common Event 190 (BirdEvent)</p>
 
 <div class="dialogue">
-    <h2>First time</h2>
-    <p class="dialogue-note">
+    <p class="dialogue-note">First time only</p>
+    
+    <p class="dialogue-note indented">
         This gets skipped if the first interaction with a bird is in ACT 3 or 4.
```

Comment:
there might be a better way to phrase this so that it can be put in just the "First time only" line hm. "first time, only if in act 2"?